package.json

```diff
@@ -25,7 +25,7 @@
   "scripts": {
     "lint": "eslint *.js  && cd example && yarn lint",
     "build": "rm -rf ./dist && tsc",
-    "test": "jest",
+    "test": "jest src/",
```

Comment:
I did this because otherwise example tests would run and you would need `detox` setted up for that